packages/flutter_tools/test/general.shard/github_template_test.dart

```diff
@@ -46,27 +57,34 @@ void main() {
       const String errorString = 'this is a 100% error';
       const String exception = 'failing to succeed!!!';
       const String doctorText = ' [✓] Flutter (Channel report';
-      FileSystem fs;
 
       setUp(() async {
         stackTrace = StackTrace.fromString('trace');
-        fs = MemoryFileSystem();
       });
 
       testUsingContext('shortened', () async {
```

Comment:
Can these be changed to `testWithoutContext`?

Reply:
Unfortunately no, building the `FlutterProject` and manifest parsing is still rife with `context`s.